influxql/iterator.go

```diff
@@ -19,7 +19,7 @@ var ErrUnknownCall = errors.New("unknown call")
 
 const (
 	// MinTime is used as the minimum time value when computing an unbounded range.
-	MinTime = int64(0)
```

Comment:
Why is the max time copied exactly and the min time decreased by one? Can you add comment to clarify why this is being done?